src/find_datalad_repos/diff.py

```diff
@@ -76,6 +77,9 @@ def main(
     for hubr in to_record.hub_datalad_org:
         if hubr.id not in old_hub_repos:
             new_record.hub_datalad_org.append(hubr)
+    for atrisr in to_record.atris:
+        if atrisr.id not in old_atris_repos:
+            new_record.atris.append(atrisr)
```

Comment:
could data structure of RepoRecord and code here be made more generic and just be done "per each RepoHost" and avoid code duplication for each possible repo?

Reply:
9284415 is the best I can come up with without upsetting mypy.